pallets/ajuna-awesome-avatars/src/lib.rs

```diff
@@ -743,7 +744,7 @@ pub mod pallet {
 			if affiliate_config.mode == AffiliateMode::Open && affiliate_config.enabled_in_buy {
 				T::FeeHandler::withdraw_and_pay_fees(
 					&buyer,
-					(),
+					WithdrawKind::Payment(()),
```

Comment:
To preserve the current behavior for AAA now, I suggest keep using `()`, this should be possible isn't it?